app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ConfirmationsController < ApplicationController
+  def create
+    @donation = find_donation
+
+    @donation.update!(confirmed: true)
+
+    flash[:success] = t(".success")
+
+    redirect_to :back
```

Comment:
I personally don't like using `:back` because it's impossible to tell where this is redirecting to, just by reading the controller. I have to know where people came from, two steps back in the app's flow, to understand this line.

There is one case where it has to be used: when multiple links come to this action.

Is that the case here? Or could this be replaced with an explicit route?